xml/grub2_yast_i.xml

```diff
@@ -171,7 +171,7 @@
      </listitem>
     </varlistentry>
     <varlistentry>
-     <term><guimenu>Enable Trusted Boot Support</guimenu>
```

Comment:
Even if the actual UI string should be written in title style (which I have not checked), nobody will kill us for changing this, I assume. OTOH, I often also used title style for a UI string in the docs although the software string was written in sentence style. As long as the docs cover what the string actually says, the spelling should not much make a difference to readers.

Reply:
The new default is ignoring all tagged content (i.e. no case changes within those). That may not be the right option, e.g. for emphasis. We could define a list of tags that should be converted though. What do you think?

Reply:
Off the top of my head I would vote for changing from title style to sentence style at least within the followings tags:
- citetitle
- guimenu
- emphasis